libraries/radwordsprocessing/formats-and-conversion/rtf/rtfformatprovider.md

```diff
@@ -32,6 +32,9 @@ The code from __Example 1__ shows how to use __RtfFormatProvider__ to import an
 #### __[C#] Example 1: Import document from a file__
 
 {{region cs-radwordsprocessing-formats-and-conversion-rtf-rtfformatprovider_0}}
+	using Telerik.Windows.Documents.Flow.FormatProviders.Rtf;
+	using Telerik.Windows.Documents.Flow.Model;
```

Comment:
Do not add usings to the snippet - it won't build when directly pasted. Instead, use full class names.